### ISSUE TITLE
Mouse disappears

### ISSUE THREAD

**Issue by G3-Logic** [avocode/avocode#2112]
Mouse constantly disappears when jumping from Brackets to Avocode. Problem goes away when I quit out of Avocode and doesn't happen while using Photoshop or Affinity Designer. Any idea why that may be happening? Very annoying. I can take a screen capture if that would help!

---
* **Version**: 3.3.2
* **Sketch plugin version**: 50.0.0
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 26220
---
```
No stack trace available
```

**Comment by jakubzitny** [comment]
Hi @G3-Logic, thanks for letting us know. Can you make a video please?

**Comment by jakubzitny** [comment]
Hi @G3-Logic, thanks for letting us know. Can you make a video please?

**Comment by G3-Logic** [comment]
Hey thanks so much for following up. For some strange reason, its not happening anymore. I used to only experience this when Sketch was open but last week Sketch was closed and the only way to stop it was to quit out of Avocode. So I assumed Avocode was having the same problem as Sketch.

I will keep an eye on it and if it starts happening again I will definitely let you know. Sorry for the trouble…

Guy Scott
--

Guy F. Scott III
G3 Logic Media Design
207.350.7242
